### ISSUE TITLE
FAQ: Add example scenario of how this license can play out in practice considering forking and contributors' copyright?

### ISSUE THREAD

**Issue by nilsnh** [ContributorCovenant/hippocratic-license#60]
Hi there. Thank you for creating license. 

I’m seriously considering using this license. I’m just wondering about how it will play out in practice when:

1. I write code licensed under this license where I have full copyright. 
1. To accept contributions, is it common to have contributors waive their copyright?

With this in mind I think this part of the license is relevant. 

> “For clarity, although Licensee is free to create modifications of the Software and distribute only the modified portion created by Licensee with additional or different terms, the portion of the Software not modified must be distributed pursuant to this License.”

If someone were to fork my project, could they then over time write more and more code under their copyright, and then finally not have to adhere to this license anymore?

I’m not asking for legal advise here, but maybe for a hypothetical scenario that demonstrates how this license will work out in practice.

**Comment by nilsnh** [comment]
I did some more research. Since this license is based on the MIT license I found some sources that might apply here:

- https://softwareengineering.stackexchange.com/a/277699
- https://gist.github.com/fbaierl/1d740a7925a6e0e608824eb27a429370
- https://writing.kemitchell.com/2016/09/21/MIT-License-Line-by-Line.html
- https://softwareengineering.stackexchange.com/a/158011
- https://opensource.stackexchange.com/a/8624
- https://writing.kemitchell.com/2017/02/16/Against-Legislating-the-Nonobvious.html

To sum up some reasoning for myself: 

- By default no contributors gives up any copyright, instead we license the code we write to each other. 
- New code by contributors can be under a different license, but the original code needs to be accompanied by the original license.
- Contributing code to a Github repository without a contributors license will automatically license the new code under the existing license [as per Github's Terms of Service](https://help.github.com/en/github/site-policy/github-terms-of-service#6-contributions-under-repository-license).
- Code licensed under HL can be used in repositories which is licensed under a different license. However, the original code needs to be accompanied by the HL license.

Hope someone will weigh in here if my reasoning is wrong, or some point needs to be made more nuanced. 

Edit: [IANAL](https://en.wikipedia.org/wiki/IANAL).

**Comment by CoralineAda** [comment]
Would you be interested in writing a  blog post for us on the topic? It could be reviewed by our IP experts.

**Comment by nilsnh** [comment]
> Would you be interested in writing a blog post for us on the topic? It could be reviewed by our IP experts.

Yes, I would be interested in writing a blog post on the topic @CoralineAda . And the topic could be: "How does the Hippocratic license play out in practice?" Something in that line, or something else?

**Comment by CoralineAda** [comment]
Something along those lines? You can brainstorm in #wg-blog

**Comment by nilsnh** [comment]
Sounds good!
